packages/vscode-graphql/src/extension.ts

```diff
@@ -76,7 +76,7 @@ export async function activate(context: ExtensionContext) {
         // TODO: load ignore
         // These ignore node_modules and .git by default
         workspace.createFileSystemWatcher(
-          '**/{*.graphql,*.graphqls,*.gql,*.js,*.mjs,*.cjs,*.esm,*.es,*.es6,*.jsx,*.ts,*.tsx,*.vue,*.svelte,*.cts,*.mts}',
+          '**/{*.graphql,*.graphqls,*.gql,*.js,*.mjs,*.cjs,*.esm,*.es,*.es6,*.jsx,*.ts,*.tsx,*.vue,*.svelte,*.cts,*.mts,*.json}',
```

Comment:
```suggestion
          '**/*.{graphql,graphqls,gql,js,mjs,cjs,esm,es,es6,jsx,ts,tsx,vue,svelte,cts,mts,json}',
```

@acao I think this can be simplified to ?

Reply:
also `esm` and `es6` extension should be removed 😅

Reply:
```suggestion
          '**/*.{graphql,graphqls,gql,js,mjs,cjs,jsx,ts,tsx,vue,svelte,cts,mts,json}',
```

Reply:
@B2o5T these unusual seeming extensions were requested by users earlier this year and last, and there were lots of emoji reacts :shrug: anecdotal expderience isn't always enough to evaluate userland haha